packages/dd-trace/test/config.spec.js

```diff
@@ -499,6 +507,7 @@ describe('Config', () => {
       site: 'datadoghq.eu',
       port: 6218,
       service: 'service',
+      extraServices: ['service1', 'service2'],
```

Comment:
this test is not needed tbh, it's just testing a priority thing with URL

Reply:
removed